DIMS-Core/DIMS-Core.BusinessLayer/Models/Task/MyTaskModel.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+
+namespace DIMS_Core.BusinessLayer.Models.Task
+{
+    public class MyTaskModel
```

Comment:
Why is it `My...` ?

Reply:
I did rename MyTask -> CurrentTask
```suggestion
    public class CurrentTaskModel